test/hack/test.yml

```diff
@@ -13,6 +13,9 @@ onboot:
     command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
   - name: check-kernel-config
     image: linuxkit/test-kernel-config:1aaef970b5f70791d74d6f980ad38af4035948f8
+  - name: sleep
+    image: linuxkit/poweroff:afe8f7dd0d47a7991c54519b0f09124cb8c4e300
+    command: ["/bin/sleep", "120"]
```

Comment:
The `poweroff` package also takes an numerical argument for sleep. Maybe this needs bumping from 3s to something a bit more

Reply:
ah yes I forgot that; lets see if this has any effect anyway and will change that instead.

Reply:
ok that did fix it, will adjust....